### DISCUSSION TITLE
Take ackee to the next level

### DISCUSSION THREAD

**Discussion by MathisBarre** [electerious/Ackee#292] [Ideas]
Recently i started using splitbee analytics and I found a lot of ideas that should be in Ackee.

A lot of things should evolve but the most importants in my opinion are :
- Add the ability to choose custom date interval for the data we are viewing (and if we can easily choose last one week, last one month etc... it can be even better) 
![image](https://user-images.githubusercontent.com/33762559/134062619-e3b49dfe-d903-4717-a9db-acb15f3d96ef.png)
![image](https://user-images.githubusercontent.com/33762559/134062727-4b7602df-f076-4e22-9f5a-f43083908b3b.png)

- Classify devices sizes by sizes interval (desktop, tablet, phone). 
- Simplify the API of ackee-tracker and add automatic SPA page change detection. With that, no need to maintain a lot of framework-specific implementation (see [splitbee.io/docs/javascript-library](https://splitbee.io/docs/javascript-library))

What do you think about that ideas ?

**Comment by electerious** [comment]
See https://github.com/electerious/Ackee/issues/39

It's definitely a feature that should be added, but it's hard to do so because of the way Ackee has been designed. The current API always returns the most recent results. All you can do is changing the interval (Weekly / Monthly / Yearly). I guess it would require a lot of changes in the API. That's why it's still not a part of Ackee.
